usl_models/setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+
+setup(
+    name="usl_models",
+    packages=find_packages(),
```

Comment:
might add an
```python
install_requires=["tensorflow"],
```
in here, alongside any other high-level dependencies you care about.

Reply:
Done.